flake.nix

```
@@ -1,21 +1,39 @@
+# Get started by following https://github.com/BitGo/bitgo-microservices/blob/develop/docs/NIX.md
 {
   description = "api-ts";
 
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs";
     flake-utils.url = "github:numtide/flake-utils";
+    pre-commit-hooks.url = "github:cachix/pre-commit-hooks.nix";
```

```suggestion
    pre-commit-hooks = {
      url = "github:cachix/pre-commit-hooks.nix";
      inputs.nixpkgs.follows = "nixpkgs";
      inputs.flake-utils.follows = "flake-utils";
    };
```

Overriding these transitive dependencies should remove the duplicate nixpkgs and flake-utils from the lockfile and improve performance

---

I reverted the pre-commit hook changes from this PR